### DISCUSSION TITLE
I have to make a button "back to top" floutant on tables

### DISCUSSION THREAD

**Discussion by JayAyAre** [filamentphp/filament#11393] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

When a table is put in 100 view mode, for example, it is easy for you not to know how to go back to the top, a floating back to top button, i need it.

![image](https://github.com/filamentphp/filament/assets/73116131/94bcb7e9-f440-4d98-ba21-0fff98e5662e)

The button should be at the bottom left. 
I ask this in case there is a way to implement it in filament tables directly using the own filament nomenclature. This way, I don't have to modify the blades directly.

i would like something like this 

![image](https://github.com/filamentphp/filament/assets/73116131/cdb74773-65cd-44f8-9306-c42497a35d54)

**Comment by leandrocfe** [comment]
Are you using this table on the custom livewire component? This button should be created on the page, not on the table I guess.

**Reply by JayAyAre** [reply]
the table is on the resource of the items, how i can put the button on the compnent?

**Comment by MNhat0308** [comment]
May be this [link](https://filamentexamples.com/project/pagination-auto-scroll-to-top) help you get some ideal
